src/templates/course.html

```diff
@@ -17,7 +17,22 @@
         </div>
         <div id="course-task-list-complete" class="list-group">
         </div>
-        {% if current_user.permissions == 2 %}
+        <div id="tasks-teaching-list author" class="list-group">
+            <li class="list-group-item active" align="center"><strong>Options</strong></li>
+            <a class="list-group-item">
+                <table width="100%">
+                    <tr class="admin author" style="display: none">
+                        <td width="35%"><b>Add/Remove TA from course</b></td>
+                        <td width="15%"><input id="TAEmail" type="text" placeholder="User's Email" /></td>
+                        <td width="5%">&nbsp</td>
+                        <td width="15%"><button id="submitAddTA" type="button" class="">Add TA</button> </td>
+                        <td width="15%"><button id="submitDeleteTA" type="button" class="">Delete TA</button> </td>
+                        <td id="addTANotification" width="15%" style="color:red; visibility: hidden">  *Incorrect</td>
+                    </tr>
+                </table>
+            </a>
+        </div>
+        {% if current_user.permissions == 50 %}
```

Comment:
Can we want to make this > 50 so that admins can archive/unarchive courses as well?